src/common/styles.js

```diff
@@ -31,19 +31,27 @@ export default StyleSheet.create({
     alignItems: 'stretch',
     marginTop: STATUSBAR_HEIGHT,
   },
+  common: {
```

Comment:
I think this should be renamed. As it is already in `/common`. Then again with name 'common' creates confusion.